tests/test-definition.yaml

```diff
@@ -49,7 +49,7 @@ tests:
   - name: kerberos
     dimensions:
       - postgres
-      - hive
+      - hive-latest
```

Comment:
The question is if Kerberos is important enough to justify testing all supported Hive versions instead of only the last version. WDYT? ;)

Reply:
Yes, agree. See 7ffaa1f.